src/EpisodeDataService/GetEpisode/GetEpisode.cs

```diff
@@ -0,0 +1,72 @@
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using System.Text;
+using System.Text.Json;
+using NHS.ServiceInsights.Data;
+using NHS.ServiceInsights.Model;
+
+namespace NHS.ServiceInsights.EpisodeDataService;
+
+public class GetEpisode
+{
+    private readonly ILogger<GetEpisode> _logger;
+    private readonly IEpisodeRepository _episodesRepository;
+
+    public GetEpisode(ILogger<GetEpisode> logger, IEpisodeRepository episodeRepository)
+    {
+        _logger = logger;
+        _episodesRepository = episodeRepository;
+    }
+
+
+    [Function("GetEpisode")]
+    public HttpResponseData Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequestData req)
+    {
+        string episodeId;
+
+        try
+        {
+            episodeId = req.Query["episodeId"];
+            if (string.IsNullOrEmpty(episodeId))
+            {
+                _logger.LogError("Episode ID is not provided.");
+                return req.CreateResponse(HttpStatusCode.BadRequest);
+            }
+
+            _logger.LogInformation("Getting Episode ID: {episodeId}", episodeId);
```

Comment:
Lines 31-38 don't need to be wrapped in a try catch block because they shouldn't ever throw an exception